src/components/Navbar.js

```diff
@@ -1,16 +1,24 @@
-import React from "react";
-import logo from "../svgs/logo";
-import { Auth } from "../modules";
-import { Grid } from "react-flexbox-grid";
+import React from 'react';
+import { Grid } from 'react-flexbox-grid';
+import logo from '../svgs/logo';
+import { Auth } from '../modules';
+import { AuthContext } from '../context/AuthContext';
+import ProfileNavbar from './ProfileNavbar';
 
 const Navbar = () => (
   <nav>
-    {/* <div className="container"> */}
     <Grid>
-      {logo}
-      <Auth />
+      <a href="/">
+        {logo}
+      </a>
+
+      <AuthContext.Consumer>
+        { value => (
+          value.access_token ? <ProfileNavbar /> : <Auth />
```

Comment:
I don't get it, which thing is a module and which is a component.
From my understanding, when there are more than components in a group for a particular task, then they are modules. Otherwise, if it's just a single component then it should go in /components.